src/utils/fluxNode.ts

```diff
@@ -147,12 +148,36 @@ export function modifyFluxNodeText(
       };
 
       copy.data.fluxNodeType = FluxNodeType.TweakedGPT;
+
+      // TODO: This shouldn't apply anymore, as a node now never arrives here being named `FluxNodeType.GPT` due to auto-label. Should we remove the logic, or adapt it?
       copy.data.label =
         copy.data.label != displayNameFromFluxNodeType(FluxNodeType.GPT)
           ? copy.data.label // Preserve custom labels if necessary.
           : displayNameFromFluxNodeType(FluxNodeType.TweakedGPT);
     }
```

Comment:
Is this safe to remove? Is it still possible to end up with a GPT (edited) node?

Reply:
yeah we can remoce